tests/test_minimal/test_tools/test_backend_and_dataset_configuration/test_helpers/test_configure_backend_defaults.py

```diff
@@ -58,3 +56,33 @@ def test_simple_time_series(
             assert written_data.compression == "gzip"
         elif backend == "zarr":
             assert written_data.compressor == numcodecs.GZip(level=1)
+
+
+@pytest.mark.parametrize("backend", ["hdf5", "zarr"])
+def test_simple_dynamic_table(tmpdir: Path, case_name: str, backend: Literal["hdf5", "zarr"]):
```

Comment:
@CodyCBakerPhD you should remove or specify `case_name` here!
```suggestion
def test_simple_dynamic_table(tmpdir: Path, backend: Literal["hdf5", "zarr"]):
```